packages/cascara/src/atoms/Flex/Flex.js

```diff
@@ -54,7 +55,7 @@ const Flex = ({
     ...getClassOptions(vAlignOptions, vAlign, 'vAlign'),
   });
 
-  return <div className={classList}>{children}</div>;
+  return <Role className={classList}>{children}</Role>;
```

Comment:
```suggestion
  return <Role {...rest} as={as} className={classList} >{children}</Role>;
```
We need to expose the `as` prop here, and spread the rest of the props. The rest spread is very important for component compatibility with `as`